packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -886,7 +919,8 @@
         }
 
         .country-dropdown {
-            min-height: 35.2rem;
+            height: 100%;
+            // min-height: 35.2rem;
```

Comment:
can you please remove this as well 😬 
Thank you

Reply:
done 👍